### PR TITLE
Use rdf-native-loader to set up express handlers

### PR DESCRIPTION
**Pull request by tpluscode**
This PR adds:

* [x] loader for middleware chain
    * [x] using the `rdf-natice-loader-code` to load the handlers
* [x] loader for SPARQL views
* [x] replaces some hydra-box terms with code vocabulary

The assumption is that any `code:impementedBy` of a SPARQL view loads an expressjs handler callback. This way multiple callbacks can easily be chained to combine multiple queries, authorisation, etc in a single Hydra operation.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,13 +8,22 @@ const ApiDocumentation = require('./lib/ApiDocumentation')
...
   options = options || {}
 
+  const loaders = createRegistry(options)
   const graph = options.graph || api
 
+  const client = new SparqlHttp({
+    endpointUrl: options.sparqlEndpointQueryUrl || options.sparqlEndpointUrl,
+    updateUrl: options.sparqlEndpointUpdateUrl,
+    fetch: rdfFetch
+  })
```
Comment (tpluscode) [inline index.js]: I moved the SPARQL client to the very top.

File: lib/SparqlView.js [hunk 2]
```diff
@@ -52,25 +34,10 @@ class SparqlView extends ApiObject {
...
-    }).then((output) => {
-      if (output) {
-        res.graph(output)
-      } else {
-        res.status(204).end()
-      }
-    }).catch(next)
+    return Promise.resolve()
+      .then(() => this._executeQuery(query, options, req, res))
+      .then(next)
```
Comment (tpluscode) [inline lib/SparqlView.js]: On second thought I conclude that this should be explicitly called in the `SparqlQuery`/`SparqlUpdate` classes

File: lib/loaders/HandlersLoader.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+  if (arg.term.termType === 'Literal') {
+    return arg.value
+  }
+
+  return arg
+}
+
+module.exports = (term, dataset, { context, loaderRegistry }) => {
+  return parseArguments(cf(dataset).node(term).out(ns.code.arguments), context, loaderRegistry)
```
Comment (tpluscode) [inline lib/loaders/HandlersLoader.js]: To be replaced by `rdf-native-loader-code` once zazuko/rdf-native-loader-code#2 is resolved

File: index.js (hunk 1 above)
Comment (bergos) [reply index.js]: Makes sense. The client should be available not only to the SPARQL code, but also to the plain JS middlewares.

**Pull request merged**